packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -51,6 +54,20 @@ module.exports = {
       });
 
       child.stdout.on("data", data => {
+        // accumulate buffer from chunks
```

Comment:
Is this not currently working the way we thought it was?

Reply:
The child process will only print the repl prompt when it is ready (has its context loaded with truffle variables). The command runner monitors the child's output, looking for that repl prompt (flag) before sending data to it